### PR TITLE
Project creation flow with CLI

### PR DESCRIPTION
**Pull request by brettimus**
This PR moves over the `create-honc-app` flow (with some modifications) to allow for bootstrapping a honc app using the `npx @fiberplane/studio@latest` command. The goal is to provide scaffolding for a user's project, beyond a generic template. 

To create a new app, you can either:

1. (recommended) run `HONC_API_KEY="abc" npx <this-pr-pkg-link> create`
2. run `npx <this-pr-pkg-link>` in a folder without any existing javascript project (no package.json, no wrangler.toml, no .fpxconfig folder)

**To get the AI bootstrapping to work, you'll need an extra HONC_API_KEY parameter.** Contact me on Slack for the details. 

## CLI Creation Flow

- Ask user for a short description of the app
- Do all the usual steps of create-honc-app
- While dependencies are downloading, make a request to an AI service that can scaffold `index.ts`, `schema.ts` and `seed.ts` according to the user's description of their app
- Add instructions on launching Fiberplane Studio to the output at the end

## honc-code-gen

This is a service deployed to Cloudflare Workers that the cli can contact to generate code based off of a user's description of their project.

It will modify the following files, depending on the description provided during the create flow

- `src/index.ts`
- `src/db/schema/ts`
- `seed.ts`

## Template Changes

See: https://github.com/brettimus/honc-it-like-its-hot

- Simplify database setup (and seeding) with a `npm run db:setup` script
- Integrate Fiberplane Studio by default
- Add a script to launch Studio (`npm run fiberplane`)

## TODO

- [ ] Move updated templates repo paths away from my personal account

### CODE REVIEW

File: api/package.json [hunk 1]
```diff
@@ -39,6 +39,13 @@
     "fpx": "bin/cli.js"
   },
   "dependencies": {
+    "@clack/core": "^0.3.4",
```
Comment (brettimus) [inline api/package.json:45]: this is an unfortunate hack - the api needs the same deps as the cli in order for the cli to work after its build is copied into the api's `dist` folder

we should probably move the cli folder inside the api, but i wanna save that for a future pr

File: cli/README.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  <img src="https://github.com/fiberplane/create-honc-app/blob/9290786147fe1efa2079899064853cf754f175e5/assets/honc.png" width="200" height="200" />
+</div>
+
+<p align="center">
+    Scaffolding CLI for creating modular data APIs using TypeScript
+</p>
+
+<div align="center">
+    <code>npm create honc-app@latest</code>
+</div
```
Comment (brettimus) [inline cli/README.md]:
```suggestion
</div>
```

File: cli/README.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  <h1>Honc Honc!</h1>
+  <img src="https://github.com/fiberplane/create-honc-app/blob/9290786147fe1efa2079899064853cf754f175e5/assets/honc.png" width="200" height="200" />
+</div>
+
+<p align="center">
+    Scaffolding CLI for creating modular data APIs using TypeScript
+</p>
+
+<div align="center">
+    <code>npm create honc-app@latest</code>
```
Comment (brettimus) [inline cli/README.md]:
```suggestion
    <code>npx @fiberplane/studio@latest create</code>
```

File: cli/README.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  🪿 **[Neon](https://neon.tech)** for a relational Postgres database  
+  🪿 **[Drizzle](https://orm.drizzle.team/)** as the ORM and migrations manager  
+  🪿 **[Cloudflare](https://workers.cloudflare.com/)** Workers for deployment hosting  
+
+## Quickstart
+
+To get started run the following command:
+
+```sh
+npm create honc-app@latest
```
Comment (brettimus) [inline cli/README.md]:
```suggestion
npx @fiberplane/studio@latest create
```

File: cli/src/actions/database/d1.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import { log } from "@clack/prompts";
+
+export function showD1SetupInstructions() {
```
Comment (brettimus) [inline cli/src/actions/database/d1.ts:3]: TODO - put this in a file and link to it, the text is usually obscured by further cli steps anyhow

File: cli/src/actions/description.ts [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import type { Context } from "@/context";
```
Comment (brettimus) [inline cli/src/actions/description.ts:1]: This is the new file I added to ask for a description of the app. This is what is ultimately used for code gen

**Review by mies — APPROVED** [review]

**Review by mies — APPROVED** [review]

**Review by mies — APPROVED** [review]

**Pull request merged**